src/test/func/malloc/malloc.cc

```diff
@@ -229,6 +246,14 @@ int main(int argc, char** argv)
     test_posix_memalign(0, align + 1, EINVAL, true);
   }
 
+#if defined(__NetBSD__)
+  for (smallsizeclass_t sc = 0; sc < NUM_SMALL_SIZECLASSES; sc++)
+  {
+    const size_t size = sizeclass_to_size(sc);
```

Comment:
It would be good to have two loops. So the src and dst sizes are different.  If they are the same, you will hit a fast path on every test.